packages/react/src/components/OverflowMenuV2/OverFlowMenuV2-test.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright IBM Corp. 2016, 2024
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { act, fireEvent, render, screen } from '@testing-library/react';
+
+import { FeatureFlags } from '../FeatureFlags';
+import { MenuItem } from '../Menu';
+import { OverflowMenuV2 } from './';
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+
+// Mocking console.warn to track deprecation warning
+jest.spyOn(console, 'warn').mockImplementation(() => {});
```

Comment:
I think we can remove this along with the commented line on top

Reply:
Thank you, removed it